lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/main/java/io/lighty/applications/rnc/module/RncLightyModule.java

```diff
@@ -172,22 +173,17 @@ public boolean close() {
         LOG.info("Stopping RNC lighty.io application...");
         boolean success = true;
         if (rncModuleConfig.getServerConfig().isEnableSwagger()
-                && this.swagger != null && !stopAndWaitLightyModule(this.swagger)) {
+                && this.swagger != null && !LightyModuleUtils.stopAndWaitLightyModule(lightyModuleTimeout,
+                DEFAULT_LIGHTY_MODULE_TIME_UNIT, this.swagger)) {
             success = false;
         }
         if (this.rncModuleConfig.getAaaConfig().isEnableAAA()
-                && this.aaaLighty != null && !stopAndWaitLightyModule(this.aaaLighty)) {
-            success = false;
-        }
-        if (this.lightyRestconf != null && !stopAndWaitLightyModule(this.lightyRestconf)) {
-            success = false;
-        }
-        if (this.lightyNetconf != null && !stopAndWaitLightyModule(this.lightyNetconf)) {
-            success = false;
-        }
-        if (this.lightyController != null && !stopAndWaitLightyModule(this.lightyController)) {
+                && this.aaaLighty != null && !LightyModuleUtils.stopAndWaitLightyModule(lightyModuleTimeout,
+                DEFAULT_LIGHTY_MODULE_TIME_UNIT, this.aaaLighty)) {
             success = false;
         }
+        success = LightyModuleUtils.stopMultipleModules(lightyModuleTimeout, DEFAULT_LIGHTY_MODULE_TIME_UNIT,
```

Comment:
This is serious issue - we have lost previous information (previous value of `success`)